cli/cmd/project/status.go

```diff
@@ -50,3 +63,21 @@ func StatusCmd(cfg *config.Config) *cobra.Command {
 
 	return statusCmd
 }
+
+func logsFormatter(jsonStr string) (string, error) {
+	res := runtimev1.ReconcileResponse{}
+	err := protojson.Unmarshal([]byte(jsonStr), &res)
+	if err != nil {
+		return "", fmt.Errorf("error in reconcileResponse logs formatting, Error %w", err)
+	}
+
+	var errors []string
+	for i := range res.Errors {
+		errors = append(errors, res.Errors[i].String())
+	}
+
+	logs := fmt.Sprintf("Errors:\n\t%s\n\n\tAffectedPaths:\n\t%s",
```

Comment:
If `len(errors) == 0`, it should not show errors

Same for affected paths

Also, it should be `Affected paths:`, not `AffectedPaths:` in the output